apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactNewEdit.js

```diff
@@ -73,6 +73,8 @@ function ContactNewEdit({ id, data, close }) {
 	const [emailError, setEmailError] = useState();
 	const [phoneError, setPhoneError] = useState();
 	const [customFieldsError, setCustomFieldsError] = useState([]);
+	const [userId, setUserId] = useState('');
+	const [currentUsername, setCurrentUsername] = useState(username);
```

Comment:
Why you need a currentUsername when you have a form controlled username? I don't get it...

Reply:
Makes sense. Removed this variable :heavy_check_mark: